sidebarsLearn.js

```diff
@@ -156,6 +156,7 @@ module.exports = {
       items: [
         'airflow-sagemaker',
         'airflow-kafka',
+        'azure-workload-identity',
```

Comment:
```suggestion
        'connections/azure-workload-identity',
```
This should fix the build error